packages/bierzo-wallet/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "bierzo-wallet",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@material-ui/core": "^3.9.3",
+    "@types/jest": "24.0.11",
+    "@types/node": "11.12.2",
+    "@types/react": "16.8.10",
+    "@types/react-dom": "16.8.3",
+    "medulas-react-components": "^0.0.0",
+    "normalize.css": "^8.0.1",
+    "react-scripts": "2.1.8",
```

Comment:
Buuuut! react-scripts should be kept here! In the end, we are using another version in other packages of the same roots. So do not move it.

Reply:
Ok, react-scripts was already in place then.